docs/src/index.tsx

```diff
@@ -17,24 +10,10 @@ import { semanticCssOverrides } from './Style'
 const mountNode = document.createElement('div')
 document.body.appendChild(mountNode)
 
-const semanticStyleOverrides = {
-  staticStyles: [semanticCssOverrides],
-}
-
 const render = NewApp =>
   ReactDOM.render(
     <AppContainer>
-      <Provider
-        theme={mergeThemes(semanticStyleOverrides, themes.teams, {
-          // adjust Teams' theme to Semantic UI's font size scheme
-          siteVariables: {
-            htmlFontSize: '14px',
-            bodyFontSize: '1rem',
-          },
-        })}
-      >
-        <NewApp />
-      </Provider>
+      <NewApp />
```

Comment:
just to clarify for myself - could you, please, suggest what was the reason of moving (and augmenting) this theme context functionality from here?

Reply:
This was also moved because of the Mobx initial implementation, but I still think the logic for the new Theme context and the Provider should reside in some component file, rather then here. What are your thoughts on this? Maybe the better approach would be creating an App component for all these rather then having them in the routes, but anyway I would like them in some component logic...

Reply:
Plus we are using state for the theme context, so we must have some component as a wrapper.

Reply:
the reason I see the initial implementation as being more semantically correct is that because `routes` file will correspond to the app-related aspects, and theming context is something that is not closely related to it. This is a subtle question and I cannot state anything here, but would rather omit these changes from the PR for now. Please, feel free to decide by yourself, though :)

Reply:
Introduced App component that will hold the logic for providing the theme context and the provider.